static/app/views/issueList/mutations/useCreateSavedSearch.tsx

```diff
@@ -0,0 +1,54 @@
+import {SavedSearch, SavedSearchType, SavedSearchVisibility} from 'sentry/types';
+import {useMutation, UseMutationOptions, useQueryClient} from 'sentry/utils/queryClient';
+import RequestError from 'sentry/utils/requestError/requestError';
+import useApi from 'sentry/utils/useApi';
+import {makeFetchSavedSearchesForOrgQueryKey} from 'sentry/views/issueList/queries/useFetchSavedSearchesForOrg';
+
+type CreateSavedSearchVariables = {
```

Comment:
nit, this is probably the more common way we name this

```suggestion
type CreateSavedSearchParams = {
```